bait-files/src/main/webapp/review.js

```diff
@@ -0,0 +1,14 @@
+function getPanels() {
+  const accordian = document.getElementsByClassName('accordion');
```

Comment:
super small change but think the variable should be "accordion" as well as in for loop

Reply:
Thanks! Will fix it.